server/channels/app/notification_test.go

```diff
@@ -170,8 +170,6 @@ func TestSendNotifications(t *testing.T) {
 	})
 
 	t.Run("replies to post created by OAuth bot should not notify user", func(t *testing.T) {
-		th := Setup(t).InitBasic()
-		defer th.TearDown()
```

Comment:
The sub-test does not need its own test setup. The setup process was causing issues with the existing test setup.